src/expression.js

```diff
@@ -53,6 +54,23 @@ function displayScopeWalkingWarning(key, scope, computeData, readOptions) {
 //!steal-remove-end
 
 // ## Helpers
+
+// ## Helpers
+
+var builtInHelpers = {
```

Comment:
Let's try to reorganize this so we don't have to enumerate the helpers here. We should be able to export the helpers so we don't have to update this list if a new helper is added.

Reply:
I was considering this earlier. If we simply export the helpers, it would include any helpers added by `registerHelper`. Can you foresee that causing problems?

Reply:
Just tested and It breaks lots of tests if we export the helpers object directly so I used `assign` to copy the built-in helpers on init.